src/ctelnet.h

```diff
@@ -110,6 +110,7 @@ const char OPT_TIMING_MARK = 6;
 const char OPT_TERMINAL_TYPE = 24;
 const char OPT_EOR = 25;
 const char OPT_NAWS = 31;
+const char OPT_NEW_ENVIRON = 39; // https://tintin.mudhalla.net/protocols/mnes/
```

Comment:
:-1: :cry: 
This is another case where Scandum has taken an existing standard and reimplemented it in a way that contradicts the original - in this case by not following the differentiation between certain standardised variables that something following the RFC would accept as coming from the OS and others that are "user variables" which are not tightly specified - or trustable. 
```suggestion
// https://www.rfc-editor.org/rfc/rfc1572.txt, bastardised form as:
// https://tintin.mudhalla.net/protocols/mnes/
const char OPT_NEW_ENVIRON = 39;
```

Reply:
Updated comments and will consider maximizing the RFC compliance through further commits